caffe2/python/optimizer.py

```diff
@@ -17,6 +17,15 @@
 
 AuxOptimizerParams = namedtuple("AuxOptimizerParams", ["local", "shared"])
 
+def _get_lr_blob_name():
```

Comment:
@lukeyeager There is some use cases where we have different optimizers with different learning rates. So we need to do this per device per optimizer class for example we can have:
`def _get_lr_blob_name(optimizer_name)` and add the optimizer_name as prefix to the lr blob name.